integration-tests/tests/rpc/creditcoin/getEvents.spec.ts

```diff
@@ -0,0 +1,44 @@
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+describe('Creditcoin RPC', (): void => {
+  let api: ApiPromise;
+
+  beforeEach(async () => {
+    process.env.NODE_ENV = 'test';
+
+    const provider = new WsProvider('ws://127.0.0.1:9944');
+
+    api = await ApiPromise.create({
+      provider,
+      rpc: {
+        creditcoin: {
+          getEvents: {
+            params: [
+              {
+                name: 'at',
+                type: 'Hash',
+                isOptional: true
+              }
+            ],
+            type: 'RpcMethodResult'
```

Comment:
Getting another error. I'm not sure what's the expected return type here.

```bash
$ yarn test tests/rpc/creditcoin/getEvents.spec.ts 
yarn run v1.22.17
$ polkadot-dev-run-test --forceExit --runInBand tests/rpc/creditcoin/getEvents.spec.ts
$ polkadot-dev-run-test --forceExit --runInBand tests/rpc/creditcoin/getEvents.spec.ts
  console.warn
    2022-03-23 19:09:42        API/INIT: RPC methods not decorated: creditcoin_eventsSubscribe, creditcoin_eventsUnsubscribe

      at apply (node_modules/@polkadot/util/logger.cjs:81:22)
      at Object.warn (node_modules/@polkadot/util/logger.cjs:161:14)
      at ApiPromise._filterRpcMethods (node_modules/@polkadot/api/base/Decorate.cjs:311:9)
      at ApiPromise._filterRpc (node_modules/@polkadot/api/base/Decorate.cjs:274:10)
      at ApiPromise._metaFromChain (node_modules/@polkadot/api/base/Init.cjs:350:10)

  console.warn
    2022-03-23 19:09:42        REGISTRY: Unable to resolve type RpcMethodResult, it will fail on construction

      at apply (node_modules/@polkadot/util/logger.cjs:81:22)
      at Object.warn (node_modules/@polkadot/util/logger.cjs:161:14)
      at TypeRegistry.getUnsafe (node_modules/@polkadot/types/create/registry.cjs:359:11)
      at TypeRegistry.get (node_modules/@polkadot/types/create/registry.cjs:344:17)
      at TypeRegistry.getOrUnknown (node_modules/@polkadot/types/create/registry.cjs:420:17)
      at Object.10 (node_modules/@polkadot/types-create/create/class.cjs:120:61)
      at constructTypeClass (node_modules/@polkadot/types-create/create/class.cjs:165:43)

  console.error
    2022-03-23 19:09:42        RPC-CORE: getEvents(at?: Hash): RpcMethodResult:: createType(RpcMethodResult):: DoNotConstruct: Cannot construct unknown type RpcMethodResult

      at apply (node_modules/@polkadot/util/logger.cjs:81:22)
      at Object.error (node_modules/@polkadot/util/logger.cjs:146:14)
      at logErrorMessage (node_modules/@polkadot/rpc-core/bundle.cjs:73:5)
      at node_modules/@polkadot/rpc-core/bundle.cjs:264:11

 FAIL  tests/rpc/creditcoin/getEvents.spec.ts
  Creditcoin RPC
    ✕ getEvents() should return some events (725 ms)

  ● Creditcoin RPC › getEvents() should return some events

    createType(RpcMethodResult):: DoNotConstruct: Cannot construct unknown type RpcMethodResult

      at createTypeUnsafe (node_modules/@polkadot/types-create/create/type.cjs:73:18)
      at TypeRegistry.createTypeUnsafe (node_modules/@polkadot/types/create/registry.cjs:323:46)
      at RpcCore._formatOutput (node_modules/@polkadot/rpc-core/bundle.cjs:413:21)
      at RpcCore._formatResult (node_modules/@polkadot/rpc-core/bundle.cjs:222:27)
      at callWithRegistry (node_modules/@polkadot/rpc-core/bundle.cjs:250:19)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        3.792 s, estimated 5 s

```